contracts/JBTiered721Delegate.sol

```diff
@@ -26,7 +27,7 @@ import './structs/JBTiered721Flags.sol';
   Votes: A helper for voting balance snapshots.
   Ownable: Includes convenience functionality for checking a message sender's permissions before executing certain transactions.
 */
-contract JBTiered721Delegate is IJBTiered721Delegate, JB721Delegate, Ownable {
+contract JBTiered721Delegate is IJBTiered721Delegate, IERC2981, JB721Delegate, Ownable {
```

Comment:
Nit: Interfaces should be inherited first (on the right), this makes it less likely we run into `linearization of inheritance graph impossible` issues later on